### ISSUE TITLE
Google Drive and file names

### ISSUE THREAD

**Issue by joep1977** [Zettlr/Zettlr#4740]
### Description

In the new version, it is not possible anymore to show the title at the top of the screen in combination with Google Drive, see below:

![image](https://github.com/Zettlr/Zettlr/assets/96820649/34ae3ebc-f8d0-410e-97be-af57136bc619)

The location of the file is G:\Mijn Drive\...\Zettlr\Theology\20231018115253.md

When I try a different Workspace that is either stored locally (C drive) of via Dropbox, the functionality works.

### Reproducing

When I create a new file, add the YAML frontmatter and name a title, it is not shown in the tab above.

### Zettlr Version

Beta (if applicable)

### Specify version

3.0.2

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 11

**Comment by joep1977** [comment]
Correction: now, for some reason, it also doesn't work with Dropbox. File path is: C:\Users\...\Dropbox\Zettlr\Theology\20231018123014.md

**Comment by joep1977** [comment]
Oddly enough, it does work if the folder of the workstation is located elsewhere. So, it works when the Zettlr folder is on Google Drive and the Workstation folder is on Dropbox...

**Comment by joep1977** [comment]
More specifically, the behavior as described [here](https://docs.zettlr.com/en/core/yaml-frontmatter/#:~:text=A%20YAML%20frontmatter%20is%20a,%2C%20keywords%2C%20and%20the%20title.) is not working: "This variable will override the filename in various places where Zettlr displays the filename (such as the file list and the document tab bar)."

**Comment by nathanlesage** [comment]
Does this still happen? I've just realized the issue is more than two years old, and Zettlr has had quite a few updates in between.

**Comment by joep1977** [comment]
No, it is fixed now. It takes approx. 30 seconds or restarting of Zettlr, but then the title is shown.

**Comment by nathanlesage** [comment]
Thank you for the response! The lengthy start up is tracked in another issue and we have it on our radar!
